Makefile.am

```diff
@@ -32,6 +32,8 @@ EXTRA_DIST   = README.md autogen.sh \
 	       win32/ctags.rc win32/ctags.exe.manifest win32/resource.h \
 	       win32/ctags_vs2013.sln \
 	       win32/ctags_vs2013.vcxproj win32/ctags_vs2013.vcxproj.filters \
+	       win32/ctags_vs2013.vcxproj.in win32/ctags_vs2013.vcxproj.filters.in \
+		   win32/Makefile \
```

Comment:
It looks the indent is different from the other lines.

Reply:
I've fixed it.
Thank you.